requirements/edx/base.txt

```diff
@@ -438,7 +438,7 @@ edx-django-release-util==1.4.0
     #   edxval
 edx-django-sites-extensions==4.2.0
     # via -r requirements/edx/kernel.in
-edx-django-utils==5.16.0
+edx-django-utils==6.0.0
```

Comment:
Version `6.0.0` dropped support for `Python 3.8` [[CHANGELOG](https://github.com/openedx/edx-django-utils/releases/tag/v6.0.0)]